### PR TITLE
Fix language dropdown in attribute creation title creation

### LINKED ISSUE #26252: BO - the language dropdown is not well displayed when resizing the window
#### Describe the bug

when we resize the window the language dropdown is not well displayed

#### Expected behavior

resize the window the language dropdown should be well displayed

#### Steps to Reproduce

1. Go to catalog >Attributes & Features > Add new attribute > resize window (the language dropdown is not ok)
2. Go to shop parameters> customer settibngs> Titles> add new social title

**Screenshots**
image
image

The issue is not present in 1.7.8.0

#### Additional information

* PrestaShop version: develop/1.7.8.x
* PHP version: 7.3

### PR DESCRIPTION
**Pull request by marwachelly**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | when we resize the window the language dropdown is not well displayed
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/26252
| How to test?      | resize the window the language dropdown should be well displayed
| Possible impacts? | /

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26253)

### CODE REVIEW

File: admin-dev/themes/default/scss/partials/_forms.scss [hunk 1]
```diff
@@ -295,6 +295,7 @@ input[type="color"]:hover,
   .translatable-field {
     .btn.dropdown-toggle {
       height: $input-height-base;
+      min-width: 62px;
```
Comment (Hlavtox) [inline admin-dev/themes/default/scss/partials/_forms.scss]: Please try to use `white-space: nowrap;`. If we change a font or it's size, your solution might no longer work.

**Review by matthieu-rolland — APPROVED** [review]
lgtm

**Comment by sowbiba** [comment]
@marwachelly you are supposed to fork the repo and create a branch on your own fork

https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/

**Review by sowbiba — COMMENTED** [review]
Contribution seems good to me. Will approve it when you reopen the PR with a fork ;)

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Hi @marwachelly I'm sorry but we have to close your pull request.
Could you please use the fork process next time?

**Comment by marwachelly** [comment]
Hi @sowbiba, @PierreRambaud,

I added a new PR: https://github.com/PrestaShop/PrestaShop/pull/26328

Thanks!

**Pull request closed without merging**